data.sql

```diff
@@ -6,3 +6,13 @@ VALUES (1, 'Agumon', '2020-2-3', 0, true, 10.23),
   (3, 'Pikachu', '2021-1-7', 1, false, 15.04),
   (4, 'Devimon', '2017-5-12', 5, true, 11);
 
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
```

Comment:
- Please, kindly ensure you're generating your IDs automatically instead of manually adding them as this might cause some errors in your database, and also, it would be hard for you to keep track of the numbers as you might make mistakes. it is best to generate them automatically. please try removing it.

Reply:
* This issue hasn't been solved i get errors when i try to run this query below is a screenshot HINT 💭 like the previous CR pointed you shouldn't manually add  The ID as according to your schema `ids` are generated automatically `id INT GENERATED ALWAYS AS IDENTITY` kindly remove the `id` column from your insert query and the integers added also 👍 
![image](https://user-images.githubusercontent.com/52497246/183856338-2b5ea722-98d1-4225-8716-3ccb351ed364.png)